Swiftris/Swiftris/GameViewController.swift

```diff
@@ -17,14 +17,22 @@ class GameViewController: UIViewController, SwiftrisDelegate, UIGestureRecognize
     var panPointReference:CGPoint? // keep track of last point when pan begins
     var player:AVAudioPlayer!
     var isPause: Bool = false
+    var timer = NSTimer()     // for countdown timer
+    var timerCount = 90       // for countdown timer
```

Comment:
This is probably something that should be user-configurable. Some users may want to play shorter games while other may want to play longer games. Let's discuss how you might go about providing this option for your users during our next call.